experiments/mjc_rfgps_example/hyperparams.py

```diff
@@ -0,0 +1,174 @@
+""" Hyperparameters for MJC peg insertion policy optimization. """
```

Comment:
Update comment to be reacher, not peg insertion. Also maybe call this rfgps_reacher_example or something. The mjc_example and mjc_mdgps_example are both peg insertion...